Updated instructions to match GCP interface (#352)
opnsense/docs: source/manual/how-tos/cloud_backup.rst

    .. image:: ./images/google_drive_enable.png

> -  On the next page that appears, click the **Credentials** item on the left-side menu:
> 
>     .. image:: ./images/google_create_credentials.png
> 

Reviewer: I think these two steps can be simplified by just directing the user directly to the service account page. So first delete the Credentials page step (the corresponding image `google_create_credentials.png` can also be deleted).

```suggestion
```

Author: Done.

> -  On the Credentials page, click the **Manage service accounts** hyperlink that appears next to the Service Account table:

Reviewer: Then direct the user to search for "service accounts" and go directly to the page. I have attached a replacement screenshot for `google_select_service_account.png`, which will need to be substituted: ![google_select_service_account](https://user-images.githubusercontent.com/53196309/133861481-99f53c35-3ad8-434b-9e48-0895b5da32ba.png)).

```suggestion
-  Now create a service account for your project. In the search bar at the top of the page, search for “service accounts" and click on the corresponding search result:

```

Author: I agree, this is a much easier way to get to this, and it should change less in the future.

-  You will be prompted to download the key. Do so, and then click **CLOSE** in the popup screen.

> 1. Set up a Google Drive folder

Reviewer: This change is not correct, as creating the Google Drive folder is the second step. The first step is creating the Google project.

```suggestion
2. Set up a Google Drive folder
```

Author: This was an unintended change that I missed when preparing the PR. I've reverted the code.
